### PR TITLE
re-added changed from pre-wipe friday work

### PR DESCRIPTION
**Pull request by TMisch**
adding back the changes that got lost on Friday. Changes include model update from dates to strings, database saving to American/Chicago timezone and logging timestamp

### CODE REVIEW

File: api/helpers/monitor.js [hunk 1]
```diff
@@ -19,17 +20,24 @@ module.exports = app => {
...
-        let date = new Date();
+        var date = new Date();
+        //var offset = date.getTimezoneOffset() / 60;
+        // console.log(offset);
+        // date.setHours(date.getHours() - offset);
+        // console.log(date);
 
         // creates new status and sets attributes
         let newStatus = {
           alternateParking: true,
```
Comment (alex-stout) [inline api/helpers/monitor.js]: Is there a way to make the `alternateParking` false?

File: api/helpers/monitor.js [hunk 2]
```diff
@@ -44,17 +52,18 @@ module.exports = app => {
...
-      ``;
       return "Even";
     } else {
       return "Odd";
     }
   }
 
   //adds 72 hours to the starting date
+  //needed to add extra hour to be a true 72 hours from start date
   function getExpirationDate(date) {
```
Comment (alex-stout) [inline api/helpers/monitor.js]: Try to use ES6 syntax

```suggestion
  getExpirationDate = date => {
```

File: api/helpers/monitor.js (hunk 1 above)
Comment (TMisch) [reply api/helpers/monitor.js]: I guess there isn't. I didn't even think about that part. The way that it works now is that it just puts a new record in the database with an expiration date. But I guess if the banner isn't there anymore I could just update the database with a record that says it's not in effect any longer.

File: api/helpers/monitor.js (hunk 2 above)
Comment (TMisch) [reply api/helpers/monitor.js]: Okay, I'll update that when I get a chance

File: api/helpers/monitor.js (hunk 1 above)
Comment (alex-stout) [reply api/helpers/monitor.js]: Yeah I guess it's not necessary, but without it it isn't very explicit whether it's active or not. It's up to you how you want to implement it!

**Pull request merged**